### PR TITLE
allow go install github.com/flashbots/mev-boost

### PR DESCRIPTION
**Pull request by metachris**
## 📝 Summary

Added a root `main.go` file and moved `cmd/mev-boost/main.go` to `cli/` (replaced with a symlink to root main.go). Allows direct installation like this:

```bash
go install github.com/flashbots/mev-boost@latest
```

---

## ✅ I have run these commands

* [x] `make lint`
* [x] `make test`
* [x] `make run-mergemock-integration`
* [x] `go mod tidy`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,25 +92,23 @@ First, install and run one of the [supported consensus clients](#consensus-clien
 
 #### Kiln testnet
 
-Here are general setup instructions for Kiln: https://notes.ethereum.org/@launchpad/kiln
```
Comment (avalonche) [inline README.md:89]: Why is the link to the setup instructions for Kiln removed?

File: cli/main.go [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+	} else if *useGenesisForkVersionMainnet {
+		genesisForkVersionHex = genesisForkVersionMainnet
+	} else if *useGenesisForkVersionKiln {
+		genesisForkVersionHex = genesisForkVersionKiln
+	} else if *useGenesisForkVersionRopsten {
+		genesisForkVersionHex = genesisForkVersionRopsten
+	} else if *useGenesisForkVersionSepolia {
+		genesisForkVersionHex = genesisForkVersionSepolia
+	} else {
+		log.Fatal("Please specify a genesis fork version (eg. -mainnet or -kiln or -ropsten or -genesis-fork-version flags)")
```
Comment (avalonche) [inline cli/main.go:91]: can update with `-sepolia`

**Review by avalonche — APPROVED** [review]

**Review by Ruteri — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (metachris) [reply README.md:89]: it's not the right place, the setup instructions are in other places

File: cli/main.go (hunk 2 above)
Comment (metachris) [reply cli/main.go:91]: nice find

**Pull request merged**